pkg/services/query/query.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/adapters"
 	"github.com/grafana/grafana/pkg/services/datasources"
 	"github.com/grafana/grafana/pkg/services/oauthtoken"
-	queryModels "github.com/grafana/grafana/pkg/services/query/models"
+	publicDashboards "github.com/grafana/grafana/pkg/services/publicdashboards/models"
```

Comment:
This alias is different from the package name, should be similar. This could be because the actual `models` name is not a good fit for this package?

Reply:
Naming is hard, lol. They're functions that work with dashboard queries within the public dashboards domain. What about `PublicDashboardQueries`?

Reply:
It is! I think it depends on whether we are keeping these in the models package or not. Did you consider adding them to the service instead of models?

Reply:
Yeah, I'm going to move them into the service package. I'll update the name to reflect that better